### ISSUE TITLE
AVT 1 - Carbon X Tile Component DAP violation

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1835]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed description
AVT 1 Run DAP on the example and there is 1 violation. See screenshot.
<img width="1230" alt="screen shot 2019-02-15 at 8 30 53 am" src="https://user-images.githubusercontent.com/21676914/52863196-d41be300-30fc-11e9-9e3f-8acee35d7c95.png">
Note: This violation can be fixed by adding an href="#" to the link in the clickable tile. I also recommend adding link text and content to the first tile.

**Comment by snidersd** [comment]
DAP was run again this afternoon and found new issues in the Selectable Tile. See screenshot attached.
<img width="1115" alt="screen shot 2019-03-04 at 11 35 44 am" src="https://user-images.githubusercontent.com/21676914/53751331-aa3f1c00-3e71-11e9-941c-5760dbcbc73e.png">
Note: Add placeholder link text to the Clickable Tile to fix the first violation and for the Selectable Tile change the label to a div if the is no visable label or add a descriptive text label to fix the remaining 2 violations.

**Comment by dakahn** [comment]
Circling back on this. Will test and figure out if we need to break these DAP errors into multiple issues.

**Comment by snidersd** [comment]
Retested with DAP the Grid tiles and Selectable tile have DAP violations.
Grid screenshot:
<img width="1295" alt="Screen Shot 2019-06-13 at 4 10 19 PM" src="https://user-images.githubusercontent.com/21676914/59467872-5c359f00-8df6-11e9-810e-53bb9f108856.png">
Selectable screenshot:
<img width="970" alt="Screen Shot 2019-06-13 at 4 09 52 PM" src="https://user-images.githubusercontent.com/21676914/59467882-62c41680-8df6-11e9-9d7e-6408adad4013.png">

**Comment by abbeyhrt** [comment]
Hi @snidersd what do you mean by Grid tiles? I also was not able to recreate any violations on SelectableTile, I checked the storybook and made a sandbox following the example in your screenshot https://2zthk.csb.app/ would you be able to verify that there are DAP violations?

**Comment by snidersd** [comment]
@abbeyhrt This issue was reported on Vanilla.

**Comment by joshblack** [comment]
Given that this is vanilla I'm going to remove it from our Carbon WCAG Compliance Epic and add it to our normal bug backlog 👍

**Comment by joshblack** [comment]
Going to close this out as a wontfix. Unfortunately, much of our attention is on the React project nowadays and the vanilla project will be moved to be community maintained as we near our next major release.
